shared/src/bundles/users/enums/users-api-path.enum.ts

```diff
@@ -1,5 +1,6 @@
 enum UsersApiPath {
     ROOT = '/',
+    GET_ONE_USER = '/user',
```

Comment:
```suggestion
    GET_USER = '/user',
```

Reply:
Fixed in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/300/commits/a53e5222a866e9a7e23099d87f575f07e895f32d](url)